src/pages/benefits/benefitFormUI/BenefitFormUI.tsx

```
@@ -63,27 +72,50 @@ const BenefitFormUI: React.FC = () => {
-            (tag: any) => tag?.descriptor?.code === "benefit_schema"
+        const schemaTag =
+          result?.responses?.[0]?.message?.catalog?.providers?.[0]?.items?.[0]?.tags?.find(
+            (tag: any) => tag?.descriptor?.code === "applicationForm"
           );
-        const resultItem = targetTag?.list?.[0]?.value;
-        const cleanedSchema = resultItem?.replace(/\\/g, "");
-        const benefit = JSON.parse(cleanedSchema) || {};
+
+        const documentTag =
+          result?.responses?.[0]?.message?.catalog?.providers?.[0]?.items?.[0]?.tags?.find(
+            (tag: any) => tag?.descriptor?.code === "eligibility"
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)